.github/workflows/test.js.yml

```diff
@@ -10,8 +10,25 @@ on:
     branches: [ master ]
 
 jobs:
-  build:
+  code:
```

Comment:
I guess we can simply add the `npm lint` before the npm test, instead of creating another  job